src/helpers/shifts.helper.ts

```diff
@@ -6,8 +6,20 @@ import { ColorHelper } from "./colors/color.helper";
 import { Colors } from "./colors/color.model";
 import { VerboseDateHelper } from "./verbose-date.helper";
 import { VerboseDate } from "../common-models/month-info.model";
+import { DataRow } from "../logic/schedule-logic/data-row";
 
 export class ShiftHelper {
+  public static getWorkersCount(day: number, data: DataRow[]): number {
+    const matrix = data.map((d) => d.rowData(false, false));
+    let count = 0;
+    for (let j = 0; j < matrix.length; j++) {
+      if (this.shiftCodeToWorkTime(matrix[j][day]) != 0) {
```

Comment:
Jeżeli zostawisz to w takiej formie to dodaj znak = :)
Zerknij sobie na to co zwraca linter: 
  Line 16:52:  Expected '!==' and instead saw '!='  eqeqeq